### PR TITLE
Fix: Remove PU name validation except for UID PU

### PR DESCRIPTION
**Pull request by sibicramesh**
--> [issue](https://github.com/aporeto-inc/aporeto/issues/1381)

### CODE REVIEW

File: monitor/remoteapi/server/server_test.go [hunk 1]
```diff
@@ -335,7 +324,7 @@ func TestCreate(t *testing.T) {
 				Name:      "^^^^",
 				Cgroup:    "/trireme/123",
 				NS:        "/var/run/docker/netns/6f7287cc342b",
-				IPs:       map[string]string{"bridge": "172.17.0.1"},
+				IPs:       map[string]string{"bridge": "172.17.0.1", "ip": "thisisnotip"},
```
Comment (t00f) [inline monitor/remoteapi/server/server_test.go:349]: You should add a unit test with utf8 characters. Creation of the PU should pass

Comment (sibicramesh) [reply monitor/remoteapi/server/server_test.go:349]: Added.

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix-pu-name",
    "component": "enforcerd",
    "pr-id": "1194",
    "commit-sha": "97235557e17b54f7fb959b0a9741dbf3cdb401ce"
  },
  {
    "project": "fix-pu-name",
    "component": "trireme-lib",
    "pr-id": "795",
    "commit-sha": "51f2c105d18d84e0efab9b22d618af4e9af58f4d"
  }
]
```

File: monitor/remoteapi/server/server_test.go [hunk 2]
```diff
@@ -115,15 +115,15 @@ func TestValidateTypes(t *testing.T) {
 			So(err, ShouldNotBeNil)
 		})
 
-		Convey("If the event name has bad charaters, it should error.", func() {
+		Convey("If the event name has utf8 charaters, it should succeed.", func() {
 			event := &common.EventInfo{
 				EventType: common.EventStart,
 				PUType:    common.ContainerPU,
-				Name:      "^^^",
+				Name:      "coolguy@gmail.com",
```
Comment (t00f) [inline monitor/remoteapi/server/server_test.go]: Can you try all the utf-8 characters here ? Check our unit tests in gaia

Comment (sibicramesh) [reply monitor/remoteapi/server/server_test.go]: Done.

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix-pu-name",
    "component": "enforcerd",
    "pr-id": "1194",
    "commit-sha": "97235557e17b54f7fb959b0a9741dbf3cdb401ce"
  },
  {
    "project": "fix-pu-name",
    "component": "trireme-lib",
    "pr-id": "795",
    "commit-sha": "28982c7b63429638183df7f306b765415335a7ca"
  }
]
```

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix-pu-name",
    "component": "enforcerd",
    "pr-id": "1194",
    "commit-sha": "47c761d36488ea2e01973b18bf17421706c25105"
  },
  {
    "project": "fix-pu-name",
    "component": "trireme-lib",
    "pr-id": "795",
    "commit-sha": "deabf1f9e2a065de06b9738f66e4248de101c786"
  }
]
```

**Review by amitlimaye — APPROVED** [review]

**Pull request merged**